backend/entityservice/views/project.py

```diff
@@ -216,8 +220,7 @@ def project_clks_post(project_id):
                 #       However, as connexion is very, very strict about input validation when it comes to json, it will always
                 #       consume the stream first to validate it against the spec. Thus the backflip to fully reading the CLks as
                 #       json into memory. -> issue #184
-
-                receipt_token, raw_file = upload_json_clk_data(dp_id, get_json(), span)
+                receipt_token, raw_file = upload_json_clk_data(dp_id, get_json(), span, uses_blocking)
```

Comment:
Another nitpick. Utility arguments such as `span` should come after the useful ones.